go/mysql/constants.go

```diff
@@ -677,3 +678,33 @@ func IsSchemaApplyError(err error) bool {
 	}
 	return false
 }
+
+type ReplicationState int
+
+const (
+	ReplicationStateRunning ReplicationState = iota
+	ReplicationStateStopped
+	ReplicationStateConnecting
+	ReplicationStateUnknown
```

Comment:
let's make `ReplicationStateUnknown` the first - hence default - value. See below